install_config/install/advanced_install.adoc

```diff
@@ -928,6 +932,20 @@ region endpoint parameter:
 openshift_hosted_registry_storage_s3_regionendpoint=https://myendpoint.example.com/
 ----
 
+[discrete]
+[[advanced-install-registry-storage-gce]]
+===== Option E: GCE Persistent Disk (gcePD)
```

Comment:
Call this 'Option E: GCS bucket storage on GCE'

We're actually configuring GCS buckets to avoid using gcePD which cannot be used by scaled registries.